server/lib/chat/chat-service.ts

```diff
@@ -48,16 +48,15 @@ export default class ChatService {
 
   constructor(
     private publisher: TypedPublisher<ChatEvent>,
-    private nydus: NydusServer,
     private userSocketsManager: UserSocketsManager,
   ) {
     userSocketsManager
       .on('newUser', (userSockets: UserSocketsGroup) => this.handleNewUser(userSockets))
-      .on('userQuit', (userName: string) => this.handleUserQuit(userName))
+      .on('userQuit', (userId: number) => this.handleUserQuit(userId))
```

Comment:
This type annotation should be unnecessary (the one above it is also unnecessary)